### ISSUE TITLE
Get-SPWeb issue

### ISSUE THREAD

**Issue by yamineT** [MicrosoftDocs/OfficeDocs-Enterprise#132]
Get-SPWeb cmdlet doesn't exist on SPO. Could you please update this article?
Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 303a91ac-84b7-6c54-2cf7-ca5be2f9007c
* Version Independent ID: f38fd76e-208e-a206-2b90-f1c818fac3af
* Content: [Why you need to use Office 365 PowerShell](https://docs.microsoft.com/en-us/office365/enterprise/powershell/why-you-need-to-use-office-365-powershell)
* Content Source: [Enterprise/powershell/why-you-need-to-use-office-365-powershell.md](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/blob/live/Enterprise/powershell/why-you-need-to-use-office-365-powershell.md)
* Service: **o365-administration**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by AndreaBarr** [comment]
@yamineT Thank you for this feedback

We appricate you taking the time to make the docs better. I am going to get this article over to the Enterprise team for investigation so we can get this updated. 

I was able to get this into the backlog for the Enterprise team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
